src/Storage/Entity/Content.php

```diff
@@ -312,6 +312,25 @@ public function setValues(array $values)
     }
 
     /**
+     * Helper to return an array of user-defined values from the Entity.
+     * This excludes meta fields set by Bolt.
+     */
+    public function getValues()
+    {
+        $contentType = $this->getContenttype();
+        if (!isset($contentType['fields'])) {
+            return [];
+        }
+
+        $fields = array_keys($contentType['fields']);
+        $allValues = $this->toArray();
+
+        return array_filter($allValues, function ($key) use ($fields) {
+            return in_array($key, $fields);
+        }, ARRAY_FILTER_USE_KEY);
```

Comment:
Might want to add a `@deprecated` around the support hack, and we'll pull it out of `master`